### DISCUSSION TITLE
Scanned PDFs

### DISCUSSION THREAD

**Discussion by Matt-Scheetz** [UglyToad/PdfPig#740] [General]
Is there, or are there plans to create, an api to determine if the PDF is scanned?

I've tested on quite a few scanned PDFs, some come back with text content of empty string, some com back with all unicode characters.

**Comment by rmarinho** [comment]
I would like to know that , and how can extract the images correctly 

example: https://bibliotecamunicipal.espinho.pt/fotos/documentos/3118_02_01_1992_16290547425fec6ead6bfb0.pdf
